lib/cryptography/utils.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Kadena.Cryptography.Utils do
+  @moduledoc """
+  Documentation for `Cryptography` utils functions.
```

Comment:
IMO to have consistency in the module doc from all modules, you should specify `Cryptography.Utils`